pallets/capacity/README.md

```diff
@@ -0,0 +1,59 @@
+# Capacity Pallet
+
+Manages the staking and balances for Capacity, an alternative payment system on Frequency.
+
+## Summary
+
+Capacity is an alternative payment system for a subset of calls that renews each Epoch.
+Tokens staked create Capacity for a targeted Provider.
```

Comment:
Do we want to link out to places in the docs where we define stake, capacity, provider, etc?

Reply:
I think it is enough with the learn more line and the content below.